src/commons/stores/cluster-store.js

```diff
@@ -296,10 +292,27 @@
                     }
                 }
             }
-            
+
             temp.hosts = store.getAssociatedHosts(cluster);
             return temp;
         }
+
+        function _getProfileStatus(temp, cluster) {
+            var status = "",
+                profileStatus = {
+                    "enabled": "Enabled",
+                    "disabled": "Disabled",
+                    "mixed": "Mixed"
+                };
+
+            if (temp.jobType === "EnableDisableVolumeProfiling" && temp.currentStatus === "in_progress") {
```

Comment:
Is there a chance that some other jobtype can come? why is this check 'temp.jobType === "EnableDisableVolumeProfiling" '

Reply:
@cloudbehl Yes. There are various jobs that can be in `current_job` like ImportCluster, UnmanageCluster.